src/renderer/sampledata_registry.json

```diff
@@ -1,133 +1,133 @@
 {
   "Carbon Model": {
     "filename": "Carbon.zip",
-    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post432%2Bgdecbeffb%2Fdata%2FCarbon.zip?generation=1622754766634004&alt=media",
+    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post721%2Bg0f8cc295%2Fdata%2FCarbon.zip?generation=1629913657838357&alt=media",
     "filesize": "1900623"
   },
   "Coastal Blue Carbon": {
     "filename": "CoastalBlueCarbon.zip",
-    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post432%2Bgdecbeffb%2Fdata%2FCoastalBlueCarbon.zip?generation=1622754766536120&alt=media",
+    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post721%2Bg0f8cc295%2Fdata%2FCoastalBlueCarbon.zip?generation=1629913657805202&alt=media",
```

Comment:
I wish there was a way to avoid repeating these in every url, but that's not a problem for this PR